Config/envoy.yaml

```diff
@@ -0,0 +1,74 @@
+admin: # 管理者画面の設定
+  access_log_path: /tmp/admin_access.log # ログを保存するファイル
+  address:
+    socket_address: { address: 0.0.0.0, port_value: 9901 } # 9901番ポートでlisten
+
+static_resources:
+  listeners:
+    - name: listener_0
+      address:
+        socket_address: { address: 0.0.0.0, port_value: 50020 } # 51020番ポートでlisten
+      filter_chains: # リクエストのフィルタリングの設定
+        - filters:
+            - name: envoy.filters.network.http_connection_manager # HTTP接続用に設計された組み込みフィルター
+              typed_config:
+                "@type": type.googleapis.com/envoy.extensions.filters.network.http_connection_manager.v3.HttpConnectionManager
+                codec_type: auto
+                stat_prefix: ingress_http # 接続マネージャの統計を発行する時に使用する人間が読める形式のprefix
+                access_log:
+                  - name: envoy.access_loggers.file
+                    typed_config:
+                      "@type": type.googleapis.com/envoy.extensions.access_loggers.file.v3.FileAccessLog
+                      path: "/dev/stdout" # ログは標準出力
+                route_config:
+                  name: local_route
+                  virtual_hosts:
+                    - name: local_service
+                      domains: ["*"] # virtual_hostのドメイン
+                      routes:
+                        - match: { prefix: "/" } # "/"がリクエストのルートと一致
+                          route:
+                            cluster: cc_to_bts
+                      cors:
+                        allow_origin_string_match: # 許可されたorigin
+                          - prefix: "*"
+                        allow_methods: GET, PUT, DELETE, POST, OPTIONS
+                        allow_headers: keep-alive,user-agent,cache-control,content-type,content-transfer-encoding,custom-header-1,x-accept-content-transfer-encoding,x-accept-response-streaming,x-user-agent,x-grpc-web,grpc-timeout
+                        max_age: "1728000" # access-control-max-age headerを指定
+                        expose_headers: custom-header-1,grpc-status,grpc-message # access-control-expose-headers headerを指定
+                http_filters:
+                  - name: envoy.filters.http.grpc_web # gRPC-Webクライアントを準拠したgRPCサーバーにブリッジできるようにするフィルター
+                  - name: envoy.filters.http.cors
+                  - name: envoy.filters.http.router
```

Comment:
Is CORS required for BTS?
Because I have an image that I set when I send a request from a browser.
(I'm afraid I don't know much about it, but..

Reply:
@mdonaka 
Deleted with [6392584](https://github.com/acompany-develop/QuickMPC-BTS/pull/7/commits/63925846ebb2c51ccf5816338efcdf462c88a758) because it was unnecessary.

Reply:
@KotaTakahashi9320 
Thank you!
Could you also check L32-L38 and L40?

Reply:
@mdonaka 
Thank you!
Deleted as it was unnecessary.

Reply:
@mdonaka 
I checked the operation with the following commands
```
grpcurl -plaintext -d '{"job_id": 1, "amount": 10, "triple_type": 1}' -rpc-header 'authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb29tX3V1aWQiOiI2NmIzM2RjOC01N2Y1LTQ2NTAtYmM5My02YjdmODVhYjc2ZjYiLCJwYXJ0eV9pbmZvIjpbeyJpZCI6MSwiYWRkcmVzcyI6IjEwLjAuMS4yMCJ9LHsiaWQiOjIsImFkZHJlc3MiOiIxMC4wLjIuMjAifSx7ImlkIjozLCJhZGRyZXNzIjoiMTAuMC4zLjIwIn1dLCJpc3MiOiJRdWlja01QQy1CVFMiLCJzdWIiOiIxMmEyNjQ0Ny0zODRkLTQzMzEtOTY1OS05MWQ3ODg2ODY0YmUiLCJhdWQiOlsiaHR0cDovL2xvY2FsaG9zdDo5MDAxIiwiaHR0cDovL2xvY2FsaG9zdDo5MDAyIiwiaHR0cDovL2xvY2FsaG9zdDo5MDAzIl0sImV4cCI6OTIyMzM3MTk3NDcxOTE3ODc1Mn0.-tnGOijC-GgXwozsN-6pwXRIo60w-j7gHQrg9fANpAk'  localhost:51020 enginetobts.EngineToBts/GetTriples
```

Reply:
Thank you! You're a help!